setup.py

```diff
@@ -1,8 +0,0 @@
-#!/usr/bin/env python
-
-from setuptools import setup
-
-setup(
-    setup_requires=['pbr'],
-    pbr=True,
```

Comment:
I think having a setup.py will not hurt. Just remove the last argument about pbr.